### PR TITLE
Loads SRE via MathJax.

### PR DESCRIPTION
**Pull request by zorkow**
This integrates SRE via MathJax directly. Changes are:
* Requires `mathjax-full` instead of `mathjax` package.
* Starts node with the `esm` flag to enable ES6 modules.
* Loads `tex-full` to have all packages available.

There is no more need to install or require `speech-rule-engine` explicitly. On the other hand it loads some MathJax components that are not necessarily useful and makes `node_module` larger.

Have a look if you prefer that.

### CODE REVIEW

**Comment by tvraman** [comment]
Threw a bunch of promise errors when run.
node -r esm math-server.js 
Math server running at port 5000

(node:163795) UnhandledPromiseRejectionWarning: Error: The bussproofs extension requires an output jax with a getBBox() method
    at Object.M.saveDocument [as item] (/home/raman/emacs/lisp/emacspeak/js/node/node_modules/mathjax-full/es5/input/tex-full.js:1)
    at FunctionList.execute (/home/raman/emacs/lisp/emacspeak/js/node/node_modules/mathjax-full/js/util/FunctionList.js:62:40)
    at x.AbstractInputJax.executeFilters (/home/raman/emacs/lisp/emacspeak/js/node/node_modules/mathjax-full/js/core/InputJax.js:42:17)
    at x.compile (/home/raman/emacs/lisp/emacspeak/js/node/node_modules/mathjax-full/es5/input/tex-full.js:1)
    at e.AbstractMathItem.compile (/home/raman/emacs/lisp/emacspeak/js/node/node_modules/mathjax-full/js/core/MathItem.js:57:39)
    at Object.renderMath (/home/raman/emacs/lisp/emacspeak/js/node/node_modules/mathjax-full/js/core/MathDocument.js:107:65)
    at RenderList.renderConvert (/home/raman/emacs/lisp/emacspeak/js/node/node_modules/mathjax-full/js/core/MathDocument.js:159:35)
    at e.AbstractMathItem.convert (/home/raman/emacs/lisp/emacspeak/js/node/node_modules/mathjax-full/js/core/MathItem.js:53:32)
    at r.AbstractMathDocument.convert (/home/raman/emacs/lisp/emacspeak/js/node/node_modules/mathjax-full/js/core/MathDocument.js:299:15)
    at Object.exports.MathJax.<computed> [as tex2mml] (/home/raman/emacs/lisp/emacspeak/js/node/node_modules/mathjax-full/js/components/startup.js:204:47)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:163795) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:163795) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by zorkow** [comment]
I shouldn't have made the last minute (untested) change from to `tex` to `tex-full`.
Indeed `bussproofs` package needs an output jax as it measures abstract bounding boxes for setting the proof trees.

`tex` alone should still work. Or alternatively, in order to do logic in emacspeak, we could load an output jax. I'll play with it a bit more.

**Comment by zorkow** [comment]
This should fix the problem. Likewise we could have gone back to `input/tex`. But `tex-full` loads all LaTeX package implemented in MathJax. So for doing logic or physics in emacspeak, this is what we need.

We can load 'tex-full` (plus output jax) from the `mathjax` package, too. But not the node version of SRE. Loading SRE separately has the advantage that it is more lightweight (i.e., the `node_modules` folder is smaller) and that we have full control over the SRE version that we want to load.

Anyway, these are the choices. Personally I am slightly in favour of the mathjax + sre variant over the mathjax-full variant.

**Comment by tvraman** [comment]
For now I think I'll load sre separately --- reason I'd still like to
see if  we can invest time into improving/revising the ACSS rules in
these times of WFH. Eventually, we can go  toward loading SRE as part
of mathjax (node_modules size is sort of in the noise in today's age
of bloatware:-)
